cli/app.go

```diff
@@ -124,11 +125,20 @@ const (
 
 	packageMetadataFlagFramework = "model_framework"
 
-	authApplicationFlagName          = "application-name"
-	authApplicationFlagApplicationID = "application-id"
-	authApplicationFlagOriginURIs    = "origin-uris"
-	authApplicationFlagRedirectURIs  = "redirect-uris"
-	authApplicationFlagLogoutURI     = "logout-uri"
+	authApplicationFlagName                 = "application-name"
+	authApplicationFlagApplicationID        = "application-id"
```

Comment:
Add a comment above these that they _will_ be deprecated once https://viam.atlassian.net/browse/APP-6993 is complete. IDK what the standard is for adding comments in a public repo of ours, will defer to someone on the SDK/Netcode team for a decision here